### ISSUE TITLE
Support for interpreting APT files with coordinated parallels

### ISSUE THREAD

**Issue by Johannes-Sahlmann** [spacetelescope/mirage#173]
When APT files include coordinated parallels, e.g. FGS imaging in parallel with NIRISS, the FGS observations should be generated by mirage as well.

**Comment by bhilbert4** [comment]
Agreed. At the moment there is no support for parallels at all, and minimal testing of APT files that contain parallel observations.

**Comment by Johannes-Sahlmann** [comment]
I am including some basic trials for this in PR #172 by trying to process Program 1087 (NIRISS + FGS parallels) with `write_observationlist.write_yaml`

**Comment by Johannes-Sahlmann** [comment]
This also touches upon some issues we discussed in the context of #74
